### PR TITLE
show Junit error body in Spyglass viewer

### LINKED ISSUE #9484: Spyglass Junit viewer should display more info on test failures
Currently, the Spyglass Junit viewer displays very minimal information about test failures. While more information is available in the build log viewer or through the link to the XML data, I think it would be helpful to make more detail available in the Junit viewer itself. Maybe clicking each test could reveal this information. (Specifically, right now the Junit viewer only shows Error.Message, which is sometimes empty. There should be a way to view Error.Body)

/cc @paulangton @krzyzacy @BenTheElder

### PR DESCRIPTION
**Pull request by ibzib**
fixes #9484 
/cc @BenTheElder @krzyzacy @paulangton 

Not sure how to make this look good (like #8970), let me know if anyone has ideas.

screenshot from 2018-09-19 16-10-17

### CODE REVIEW

**Comment by krzyzacy** [comment]
wonder how it looks like with a larger blob of errors

File: prow/cmd/deck/static/template/spyglass-template.html [hunk 1]
```diff
@@ -126,6 +126,15 @@
         document.getElementById(skipID).style.display = "none";
       }
 
+      function toggleExpansion(bodyId, expanderId) {
```
Comment (paulangton) [inline prow/cmd/deck/static/template/spyglass-template.html]: I don't love the idea of leaking viewer-specific functionality into the main spyglass template.

Comment (ibzib) [reply prow/cmd/deck/static/template/spyglass-template.html]: Absolutely agree, thanks for linking the issue.

Comment (paulangton) [reply prow/cmd/deck/static/template/spyglass-template.html]: That issue is a big one, I've started working on a fix for it but still ironing out the kinks

**Pull request merged**